app/lib/zeebe-api/zeebe-api.js

```diff
@@ -311,6 +330,16 @@ class ZeebeAPI {
     options = await this._withTLSConfig(url, options);
     options = this._withPortConfig(url, options);
 
+    this._log.debug('creating client', {
+      url,
+      options: filterRecursive(options, [
+        'clientId:secret',
+        'clientSecret:secret',
+        'customRootCert:blob',
+        'rootCerts:blob'
+      ])
```

Comment:
I think this list is complete. I checked `zeebe-node` types to make sure of that.